src/donationswap.py

```diff
@@ -635,21 +635,24 @@ def get_match(self, secret):
 		if my_offer is None or their_offer is None:
 			return None
 
+		myGiftAidMultiplier = (my_offer.country.gift_aid / 100.0) + 1
+		theirGiftAidMultiplier = (their_offer.country.gift_aid / 100.0) + 1
```

Comment:
I'm sure the math is sound, but shouldn't a _multiplier_ be used for multiplication, not division?
Maybe it would be better to use `myGiftAidMultiplier = 1 / (1 + my_offer.country.gift_aid / 100)` and then actually multiply the various `offer.amount`s with it?

Reply:
ack, good point, i made a bug, will fix...